persistent/tests/test_persistence.py

```diff
@@ -586,6 +666,9 @@ def test___getattribute___p__names(self):
         for name in NAMES:
             getattr(inst, name)
         self._checkMRU(jar, [])
+        # _p_mtime is special, it activates the object
+        getattr(inst, '_p_mtime')
+        self._checkMRU(jar, [OID])
```

Comment:
@jimfulton can you recall why `_p_mtime` for a ghost is special, compared to the other `_p_` attributes?